GitUI/UserControls/BlameControl.cs

```diff
@@ -153,10 +156,28 @@ private void BlameAuthor_MouseMove(object sender, MouseEventArgs e)
                 _tooltipCommit = blameCommit;
                 _lastTooltipX = newTooltipX;
                 _lastTooltipY = newTooltipY;
-                blameTooltip.Show(blameCommit.ToString(), this, newTooltipX, newTooltipY);
+                blameTooltip.Show(GetBlameToolTipText(blameCommit, _gitRevisionSummaryBuilder), this, newTooltipX, newTooltipY);
```

Comment:
GitHub has lost my main comment... 

I don't see a good way of achieving the desired - each approach has limitations. @gerhardol suggestion is probably the lesser evil of all, e.g.
```suggestion
                blameTooltip.Show(blameCommit.ToString(_gitRevisionSummaryBuilder.BuildSummary), this, newTooltipX, newTooltipY);
```

```diff
diff --git a/GitCommands/Git/GitBlame.cs b/GitCommands/Git/GitBlame.cs
index d700f0a2e..6985ea707 100644
--- a/GitCommands/Git/GitBlame.cs
+++ b/GitCommands/Git/GitBlame.cs
@@ -61,7 +61,9 @@ public GitBlameCommit(ObjectId objectId, string author, string authorMail, DateT
             FileName = fileName;
         }
 
-        public override string ToString()
+        public override string ToString() => ToString(summaryBuilder: null);
+
+        public string ToString(Func<string, string>? summaryBuilder)
         {
             StringBuilder s = new();
 
@@ -74,7 +76,16 @@ public override string ToString()
             }
 
             s.Append("Commit hash: ").AppendLine(ObjectId.ToShortString());
-            s.Append("Summary: ").AppendLine(Summary);
+
+            if (summaryBuilder is not null)
+            {
+                s.Append("Summary: ").AppendLine(summaryBuilder(Summary));
+            }
+            else
+            {
+                s.Append("Summary: ").AppendLine(Summary);
+            }
+
             s.AppendLine();
             s.Append("FileName: ").Append(FileName);
```